pkgs/development/tools/bbin/default.nix

```
@@ -0,0 +1,37 @@
+{ lib
+, stdenv
```

```suggestion
, stdenvNoCC
```

because no C compilation occurs here.